backend/pkg/console/consumer_group_offsets.go

```diff
@@ -63,13 +63,19 @@ func (s *Service) getConsumerGroupOffsets(ctx context.Context, groups []string)
 	// 1. Fetch all Consumer Group Offsets for each Topic
 	offsets, err := s.kafkaSvc.ListConsumerGroupOffsetsBulk(ctx, groups)
 	if err != nil {
-		s.logger.Error("failed to list consumer group offsets in bulk", zap.Error(err))
-		return nil, fmt.Errorf("failed to list consumer group offsets in bulk")
+		s.logger.Error("failed to list consumer group offsets", zap.Error(err))
+		return nil, fmt.Errorf("failed to list consumer group offsets")
 	}
 
 	offsetsByGroup := make(map[string]map[string]partitionOffsets) // GroupID -> TopicName -> partitionOffsets
-	for group, offset := range offsets {
-		offsetsByGroup[group] = convertOffsets(offset)
+	for _, grpOffsetRes := range offsets.Groups {
+		if kerr.ErrorForCode(grpOffsetRes.ErrorCode) != nil {
```

Comment:
should we assign a new error here for logging?

```go
if err := kerr.ErrorForCode(grpOffsetRes.ErrorCode); err != nil {
  s.logger.Warn("failed to fetch offset for consumer group",
    zap.String("group_id", grpOffsetRes.Group),
    zap.Error(err)) // logs the actual error from grpOffsetRes.ErrorCode() rather than the previous one
    continue
}
```